components/GlobalStyles.js

```diff
@@ -11,7 +11,7 @@ export const GlobalStyle = createGlobalStyle`
     background-repeat: no-repeat;
     background-size: cover;
     background-attachment: fixed;
-    margin: 80px 10px;
+    margin: 70px 10px;
```

Comment:
These are what we would call "magic numbers". At first glance, someone else might not realize what they do, and what they're for. They'd have to fiddle with these numbers to truly understand their purpose. What you can do instead (if you want to) is declare these numbers as variables, same as you have with your colors. Something like `spacing-top`, etc. :)

Reply:
Sidenote, and something to consider: perhaps you'd rather use `rem` for your font sizes. `rem` is the root "em" (a term borrowed from print/typography) of the browser, which is usually 16px. This root font size is targeted when setting a different globals size for fonts. If a person has sight issues, they'd probably prefer bigger text. If you're using `rem`, your font sizes will be adjusted automatically.

For the same reason it can be a dodgy maneuver to use `rem` for padding, margin, etc. (Which I sometimes still do, haha.)